Sources/ArcGISToolkit/Components/Augmented Reality/Utilities/ARSwiftUIView.swift

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#if !os(visionOS)
```

Comment:
Instead conditionally compiling this code, could `ARSwiftUIView` be marked as unavailable on visionOS?

Reply:
I thought about that and that is an option but you will still need the compiler directives so I thought since this type is internal we can just mark out the whole thing with the compiler directive to make things easier.